src/lib/common/Format.h

```diff
@@ -31,6 +31,15 @@
 
```

Comment:
Yes, that is what was meant with the phrase 'notifyFormat now fully working' ...